src/Sylius/Bundle/AdminApiBundle/Resources/config/app/config.yml

```diff
@@ -24,3 +24,12 @@ jms_serializer:
             sylius-core:
                 namespace_prefix: "Sylius\\Component\\Core"
                 path: "@SyliusAdminApiBundle/Resources/config/serializer"
+            sylius-addressing:
+                namespace_prefix: "Sylius\\Component\\Addressing"
```

Comment:
This solution will not work because you have overridden the previous serialization config. We need to either move the whole mapping for these classes to AdminApiBundle, create children class, just to add some mapping(which sucks) or you can check if it will work if you will change a key for this config(`sylius-addressing` -> `sylius-admin-api-addressing` or something similar)